src/config/swagger.config.ts

```diff
@@ -3,8 +3,8 @@ import { INestApplication } from '@nestjs/common';
 
 export function setupSwagger(app: INestApplication): void {
   const config = new DocumentBuilder()
-    .setTitle('Users')
-    .setDescription('The Users API description')
+    .setTitle('Leftovers API')
+    .setDescription('The Leftovers API description')
     .setVersion('0.1')
```

Comment:
You can sync displayed version here with the version in package.json:

1. Add `"resolveJsonModule": true` in `tsconfig.json` under `compilerOptions`
2. Add `import { version } from '../../package.json';`
3. Replace `.setVersion('0.1')` with `.setVersion(version)`